components/nav.js

```diff
@@ -29,6 +29,46 @@ const Navbar = glamorous.nav(({top, theme: {mediaQueries}}) => ({
     flexDirection: 'column',
   },
 }))
+// Abstract input primitives to a higher level at a later date
+const inputBase = (props, {colors}) => ({
+  width: 120,
+  borderRadius: 50,
+  paddingTop: 4,
+  paddingBottom: 4,
+  paddingLeft: 30,
+  paddingRight: 20,
+  backgroundImage: 'url(/static/images/search.svg)',
```

Comment:
Instead of making this external, would it be possible to just import the svg directly? We have a babel plugin that will give us a react component when we import an svg file.